lib/cli.js

```diff
@@ -38,6 +41,11 @@ async function main(cli, { policyDetails, setupProjectFn }) {
                 `The dependency ${dep} was found multiple times in the config file. Please refer Rules section in configuration.md`,
               );
             }
+            if (ignoredDependencies.includes(dep)) {
+              throw new Error(
+                `The dependency ${dep} was found in ignoredDependencies and custom configuration. Please refer Rules section in configuration.md`,
```

Comment:
this is fine for now, but once https://github.com/stefanpenner/supported/pull/36 lands, we should create a shortend url for it and actually show a url here for people to lookup